examples/config-ini/src/lib/config.sh

```diff
@@ -17,11 +17,12 @@ config_init() {
   [[ -f "$CONFIG_FILE" ]] || touch "$CONFIG_FILE"
 }
 
-# Get a value from the config
+# Get a value from the config.
```

Comment:
This seems to have been an older version of this file, I copied from lib and all these changes showed up.

(This is a more general question I have about how bashly handles library changes given that it's just copying them in, but I think that's a separate issue.)